### PR TITLE
Adds orbiter page

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/310

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: node-operators/networks/collators/orbiter.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    ```
+
+=== "Moonriver"
+    ```
+    {{ networks.moonriver.orbiter.max_collators }} Program Collators
+    ```
+
+=== "Moonbase Alpha"
+
+    {{ networks.moonbase.orbiter.max_collators }} Program Collators
```
Comment (eshaben) [inline node-operators/networks/collators/orbiter.md:103]:
```suggestion
    ```
    {{ networks.moonbase.orbiter.max_collators }} Program Collators
    ```

```

File: node-operators/networks/collators/orbiter.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+## 计划时长 {: #duration }
+
+目前计划仍然处于试验阶段，Moonbeam基金会将会根据执行结果实时进行调整。因此，目前并没有明确的结束日期，但整个试验将有可能在计划实施期间结束或是变更。Moonbeam基金会鼓励参与者在整个计划中提供反馈，并注意细节或概念将有可能与此处解释不同。
+
+## 参与资格 {: #eligibility }
+
+要参与Orbiter计划，您必须符合以下参与条件：
+
+- 由于计划本身要求，每个Orbiter必须通过KYC检查且不能为特定国家的居民
```
Comment (eshaben) [inline node-operators/networks/collators/orbiter.md:24]: just need to swap KYC for identity verification

File: node-operators/networks/collators/orbiter.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+- 每个个体（个人或是团体）仅能在各个网络上运行一个Oribter，如一个在Moonriver，一个在Moonbeam
+- Orbiter将无法在其Orbiter相同的网络上运行另外一个收集人账户。然而，在仅在单一网络上运行的情况下，Orbiter将能够在另外一个网络上运行收集人，如在Moonbeam上运行一个收集人，在Moonrriver上运行一个Orbiter
+
+## 沟通交流 {: #communication }
+
+Moonbeam基金会已为此计划创建一个私人的Discord群组，大部分的对话将会在其频道中进行或是以DM进行。当您填写并提交申请表单，您将会被添加至群组当中。
+
+## 申请和加入过程 {: #application-and-onboarding-process }
+
+要加入Orbiter计划，您将需要填写申请表单并提交联系方式、社群媒体账号以及收集人和节点详情。在表格最后，您将需要根据操作流程完成KYC。
```
Comment (eshaben) [inline node-operators/networks/collators/orbiter.md:35]: just need to swap KYC for identity verification

File: node-operators/networks/collators/orbiter.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+<div class="button-wrapper">
+    <a href="https://docs.google.com/forms/d/e/1FAIpQLSewdSAFgs0ZbgvlflmZbHrSpe6uH9HdXdGIL7i07AB2pFgxVQ/viewform" target="_blank" class="md-button">Moonbeam Orbiter Program Application</a>
+</div>
+当您通过了KYC并成功通过此次计划的申请，您将会收到通知并开始进行加入流程，以下为加入流程的步骤：
```
Comment (eshaben) [inline node-operators/networks/collators/orbiter.md:40]: just need to swap KYC for identity verification

**Review by eshaben — CHANGES_REQUESTED** [review]
just a small change needed! other than that it looks good!

**Comment by hyd628** [comment]
Thanks @eshaben , incorporated the feedback.

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**